### PR TITLE
[fix]flink home update err no err notify

### PR DESCRIPTION
**Pull request by monrg**
### What problem does this PR solve?
fixed flink home update err no err notify

Problem Summary:
flink home update err no err notify
image

### CODE REVIEW

File: streamx-console/streamx-console-webapp/src/views/flink/setting/View.vue [hunk 1]
```diff
@@ -357,14 +357,22 @@ export default {
...
                 }).then((resp)=>{
-                  this.flinkFormVisible = false
-                  this.$swal.fire({
-                    icon: 'success',
-                    title: values.flinkName.concat(' update successful!'),
-                    showConfirmButton: false,
-                    timer: 2000
-                  })
-                  this.handleFlinkAll()
+                  if(resp.data) {
```
Comment (wolfboys) [inline streamx-console/streamx-console-webapp/src/views/flink/setting/View.vue]: code format...

File: streamx-console/streamx-console-webapp/src/views/flink/setting/View.vue [hunk 2]
```diff
@@ -357,14 +357,22 @@ export default {
...
+                    this.flinkFormVisible = false
+                    this.$swal.fire({
+                      icon: 'success',
+                      title: values.flinkName.concat(' update successful!'),
+                      showConfirmButton: false,
+                      timer: 2000
+                    })
+                    this.handleFlinkAll()
+                  }else {
```
Comment (wolfboys) [inline streamx-console/streamx-console-webapp/src/views/flink/setting/View.vue]: code format...

**Review by wolfboys — COMMENTED** [review]
need to improve code format

File: streamx-console/streamx-console-webapp/src/views/flink/setting/View.vue (hunk 1 above)
Comment (monrg) [reply streamx-console/streamx-console-webapp/src/views/flink/setting/View.vue]: has been reformatted

**Review by wolfboys — COMMENTED** [review]
LGTM.

**Pull request merged**